src/services/measure.service.js

```diff
@@ -289,20 +291,30 @@ const evaluateMeasureForPopulation = async (args, { req }) => {
     } else {
       patients = await findResourcesWithQuery({}, 'Patient');
     }
-    patientBundles = patients.map(async p => {
-      return getPatientDataCollectionBundle(p.id, dataReq.results.dataRequirement);
-    });
+    patientIds = patients.map(p => p.id);
   }
-  patientBundles = await Promise.all(patientBundles);
-  const { periodStart, periodEnd } = req.query;
-  const { results } = await Calculator.calculateMeasureReports(measureBundle, patientBundles, {
-    measurementPeriodStart: periodStart,
-    measurementPeriodEnd: periodEnd,
-    reportType: 'summary'
-  });
 
-  logger.info('Successfully generated $evaluate-measure report');
-  return results;
+  // count number of patientIds, if over threshold, then do them with workers, otherwise do it here
+  if (patientIds.length > process.env.SCALED_EXEC_THRESHOLD) {
```

Comment:
Should we guard against a case where a user _doesn't_ want to enable the scalable calculation at all? I tried setting `EXEC_WORKERS` to 0 so the process won't work, but it still adds the jobs because the patient ID count is above the threshold.

Idk if it makes sense to introduce a new environment variable or not, but maybe we can just only do this if EXEC_WORKERS is greater than 0? Lmk what you think

Reply:
Implemented the EXEC_WORKERS > 0 check in f429040.